src/views/ProfileView.vue

```diff
@@ -254,12 +254,15 @@
   const twoFactorAuthError: ComputedRef<string> = computed(() => {
     // Early return if loading
     if (twoFactorAuthenticationStore.loading) return '';
-    const ignoredErrorCodes: string[] = ['SOFTWARE_TOKEN_VERIFICATION_ERROR', 'OTP_NICHT_GUELTIG'];
-    if (twoFactorAuthenticationStore.errorCode && !ignoredErrorCodes.includes(twoFactorAuthenticationStore.errorCode)) {
-      return t('admin.person.twoFactorAuthentication.errors.connection');
+
+    switch (twoFactorAuthenticationStore.errorCode) {
+      case 'TOKEN_STATE_ERROR':
+        return t('admin.person.twoFactorAuthentication.errors.tokenStateSelfServiceError');
+      case 'PI_UNAVAILABLE_ERROR':
```

Comment:
im Store sehe Ich           

```
if (error.response?.data.i18nKey === 'PI_UNAVAILABLE_ERROR') {
            this.errorCode = 'HARDWARE_TOKEN_SERVICE_FEHLER';
          }
```
Heisst das nicht, dass der error PI_UNAVAILABLE_ERROR eigentlich nie bei uns vorkommt, weil es immer ersetzt wird?

Reply:
Natürlich kommt der vor. Der wird nur in diesem spezifischen Fall ersetzt, weil sich sonst das ganze PopUp geschlossen hätte. (PopUp wird nur angezeit wenn der Fehler nicht gleich TOKEN_STATE_ERROR oder PI_UNAVAILABLE_ERROR ist)